### PR TITLE
autopep8

### PR DESCRIPTION
**Pull request by mlautman**
Results of running 
`autopep8 --max-line-length 130 --in-place $1` on each `.py` file

### CODE REVIEW

File: src/rqt_topic/topic_info.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def message_callback(self, message):
             self.timestamps.append(rospy.get_time())
 
             # FIXME: this only works for message of class AnyMsg
-            #self.sizes.append(len(message._buff))
+            # self.sizes.append(len(message._buff))
```
Comment (brawner) [inline src/rqt_topic/topic_info.py]: Commented code

File: src/rqt_topic/topic_widget.py [hunk 2]
```diff
@@ -360,7 +364,7 @@ def set_selected_topics(self, selected_topics):
         @type selected_topics: []
         """
         rospy.logdebug('set_selected_topics topics={}'.format(
-                                                         len(selected_topics)))
+            len(selected_topics)))
```
Comment (brawner) [inline src/rqt_topic/topic_widget.py]: Don't break line

**Comment by mlautman** [comment]
@brawner ready for merge?

**Comment by brawner** [comment]
@mlautman It's not letting me approve it, perhaps because of the failing test? It looks good to me though

**Pull request merged**